gitlab-rc.json

```diff
@@ -0,0 +1,52 @@
+{
+	"GitLab CE": {
+		"containers": {
+			"gitlab-ce": {
+				"image": "gitlab/gitlab-ce",
+				"launch_order": 1,
+				"ports": {
+					"80": {
+						"host_default": 8080,
+						"protocol": "tcp",
+						"label": "HTTP WebUI port",
+						"description": "GitLab WebUI port.Suggested default: 8080",
+						"ui":true
+					},
+					"443": {
+						"host_default": 8443,
+						"protocol": "tcp",
+						"label": "HTTPS WebUI port",
+						"description": "GitLab WebUI port.Suggested default: 8443. Not enabled in docker image by default"
+					    },
+					"22": {
+						"host_default": 2222,
+						"protocol": "tcp",
+						"label": "SSH Gitlab port",
+						"description": "GitLab SSH port. Suggested default: 2222"
+					}
+				},
+			"volumes": {
+				"/var/opt/gitlab": {
+					"description": "Choose a Share for GitLab repositories. Eg: create a Share called gitlab-repos for this purpose alone.",
+					"label": "Repository Storage",
+					"min_size": 1048576
+				},
+				"/etc/gitlab": {
+					"description": "Choose a Share for GitLab configuration. Eg: create a Share called gitlab-config for this purpose alone.",
+					"label": "Config Storage"
+				},
+				"/var/log/gitlab": {
+                                        "description": "Choose a Share for GitLab log files. Eg: create a Share called gitlab-logs for this purpose alone.",
+                                        "label": "Config logs"
```

Comment:
Looks like a comma is missing in line 40 and the indentation is off (tab vs spaces).